src/utils/lang-utils.ts

```diff
@@ -34,3 +34,22 @@ export const loadIncontextTranslation = (
     document.head.appendChild(jipt);
   }
 };
+
+/**
+ * Filter out unsupported languages and return an Object containing language code and language name
+ * @param excludedLanguages
+ * @returns Object containing language code and language name
+ */
+export const getAllowedLanguages = (
```

Comment:
Ahould we specify the platform as a prop to "getAllowedLanguages" function?

Reply:
I think the platform can handle that inside the Platform codebase. We can have same  function called with separate params for different environments as well. This was library function will be pure